src/Utilities/Compiler/Analyzer.Utilities.projitems

```diff
@@ -65,14 +65,9 @@
     <Compile Include="$(MSBuildThisFileDirectory)Extensions\WellKnownDiagnosticTagsExtensions.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)Index.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)IsExternalInit.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)Lightup\INegatedPatternOperationWrapper.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)Lightup\IOperationWrapper.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)Lightup\ITypeSymbolExtensions.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)Lightup\IFunctionPointerInvocationOperationWrapper.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)Lightup\LightupHelpers.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)Lightup\NullableAnnotation.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)Lightup\NullableContext.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)Lightup\NullableContextExtensions.cs" />
```

Comment:
Removed files moved to PublicApiAnalyzers project, which is still on MS.CA 1.2.1